tests/e2e/conftest.py

```diff
@@ -25,6 +31,163 @@
+            mcg_obj=mcg_obj_session,
+        )
+
+        # Stop MCG reconcilation
+        ocp_storagecluster_obj = OCP(
+            namespace=constants.OPENSHIFT_STORAGE_NAMESPACE, kind="storagecluster"
+        )
+        params = '{"spec": {"multiCloudGateway": {"reconcileStrategy": "ignore"}}}'
+        ocp_storagecluster_obj.patch(
+            resource_name="ocs-storagecluster", params=params, format_type="merge"
+        )
+        logger.info("Stopped MCG reconcilation!")
+
+        # Stop the NooBaa Service before restoring the NooBaa DB. There will be no object service after this point
+        ocp_deployment_obj = OCP(
+            kind="deployment", namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+        )
+        nb_operator_dc = Deployment(
+            **ocp_deployment_obj.get(resource_name="noobaa-operator")
+        )
+        nb_operator_dc.scale(replicas=0)
+        nb_endpoint_dc = Deployment(
+            **ocp_deployment_obj.get(resource_name="noobaa-endpoint")
+        )
+        nb_endpoint_dc.scale(replicas=0)
+        modify_statefulset_replica_count(
+            statefulset_name=constants.NOOBAA_CORE_STATEFULSET, replica_count=0
+        )
+        logger.info("Stopped the noobaa service!")
+
+        # Login to the NooBaa DB pod and cleanup potential database clients to nbcore
+        query = "SELECT pg_terminate_backend (pid) FROM pg_stat_activity WHERE datname = 'nbcore';"
+        noobaa_db_pod.exec_cmd_on_pod(
+            command=f'psql -h 127.0.0.1 -p 5432 -U postgres -c "{query}"'
+        )
+        logger.info("Cleaned up potential database clients to nbcore!")
+
+        # Restore DB from a local folder
+        OCP(namespace=constants.OPENSHIFT_STORAGE_NAMESPACE).exec_oc_cmd(
+            command=f"cp ./mcg.bck {noobaa_db_pod.name}:/tmp/test.db"
+        )
+        noobaa_db_pod.exec_cmd_on_pod(command="pg_restore -d nbcore /tmp/test.db -c")
+        logger.info("Restored DB from the local folder!")
+
+        # Delete secrets and restore them from a local folder.
+        # Please note that verify that there are no errors before you proceed to the next steps.
+        for secret in secrets_obj:
+            secret.delete()
+        logger.info("Deleted current secrets!")
+        for secret in secrets_obj:
+            secret.create()
+        logger.info("Restored old secrets!")
+
+        # Restore MCG reconciliation
+        params = '{"spec": {"multiCloudGateway": {"reconcileStrategy": "manage"}}}'
+        ocp_storagecluster_obj.patch(
+            resource_name="ocs-storagecluster", params=params, format_type="merge"
+        )
+        logger.info("Restored MCG reconcilation!")
+
+        # Start the NooBaa service
+        nb_operator_dc.scale(replicas=1)
+        nb_endpoint_dc.scale(replicas=1)
+        modify_statefulset_replica_count(
+            statefulset_name=constants.NOOBAA_CORE_STATEFULSET, replica_count=1
+        )
+        logger.info("Started noobaa services!")
```

Comment:
can this message be more clear on what are the noobaa services that are started?